test/test_addcall.c

```diff
@@ -28,7 +28,7 @@
 // OBJECT ../src/zone_nr.o
 
 void addmult() {}
```

Comment:
Perhaps an argument with type `qso_t` missing from here. (But not relevant as I see)

Reply:
Good point. gcc 8.3.0 gives no warning, or at least I didn't see it.
Will push the fix and then merge.